### PR TITLE
Bz1387398

### PR DESCRIPTION
**Pull request by lnykryn**

### CODE REVIEW

File: man/systemd.exec.xml [hunk 1]
```diff
@@ -740,6 +740,35 @@
...
+        once in which case the ambient capability sets are merged.
+        If the list of capabilities is prefixed with <literal>~</literal>, all
+        but the listed capabilities will be included, the effect of the
+        assignment inverted. If the empty string is
+        assigned to this option, the ambient capability set is reset to
+        the empty capability set, and all prior settings have no effect.
+        If set to <literal>~</literal> (without any further argument), the
+        ambient capability set is reset to the full set of available
+        capabilities, also undoing any previous settings. Note that adding
+        capabilities to ambient capability set adds them to the process's
```
Comment (jsynacek) [inline man/systemd.exec.xml:788]: It should probably be "process'", not "process's". Cosmetic issue.

File: src/test/test-execute.c [hunk 2]
```diff
@@ -167,6 +208,8 @@ int main(int argc, char *argv[]) {
                 test_exec_environment,
```
Comment (jsynacek) [inline src/test/test-execute.c]: Weird. Isn't ```test_exec_passenvironment``` supposed to be in the patch context in here?

**Review by jsynacek — COMMENTED** [review]
Otherwise, LGTM.

File: man/systemd.exec.xml (hunk 1 above)
Comment (lnykryn) [reply man/systemd.exec.xml:788]: Seems to be in upstream as well.

File: src/test/test-execute.c (hunk 2 above)
Comment (lnykryn) [reply src/test/test-execute.c]: Fixed, this branch was based on code before test_exec_passenvironment was added.

**Pull request merged**